app/src/routes/routesMap.js

```diff
@@ -78,6 +79,8 @@ export default {
   API_PAGE: '/api',
 
   [PROJECTS_PAGE]: '/administrate/projects',
+  [PROJECT_DETAILS_PAGE]:
+    '/administrate/project-details/:projectId/:projectSection(settings|members|events)?',
```

Comment:
May be better to define the next path: `/administrate/projects/:projectId/:projectSection(settings|members|events)?`
to be more apparent (as extend `PROJECTS_PAGE`)